default/stringtables/en.txt

```diff
@@ -6238,6 +6241,40 @@ Opinions of Empires:
 OPINIONS_OF_OTHER_SPECIES
 Opinions of Other Species:
 
+# Species Pedia Categories
+
+LITHIC_SPECIES_CLASS
+Lithic Species
+
+LITHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia LITHIC_SPECIES_TITLE]] species:
+
+ORGANIC_SPECIES_CLASS
+Organic Species
+
+ORGANIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ORGANIC_SPECIES_TITLE]] species:
+
+PHOTOTROPHIC_SPECIES_CLASS
+Phototrophic Species
+
+PHOTOTROPHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia PHOTOTROPHIC_SPECIES_TITLE]] species:
+
+ROBOTIC_SPECIES_CLASS
+Robotic Species
+
+ROBOTIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ROBOTIC_SPECIES_TITLE]] species:
+
+SELF_SUSTAINING_SPECIES_CLASS
```

Comment:
See also other similar cases.

![ship part](https://cloud.githubusercontent.com/assets/11386908/24057132/9fe685a0-0b46-11e7-9158-8164dbb2c044.png)

I guess that's a page about ship parts then...?